modules/gitops-configuring-workloads.adoc

```diff
@@ -1,12 +1,11 @@
 // Module is included in the following assemblies:
 //
-// * cicd/gitops/configuring-resource-quota.adoc
+// * managing_resource/configuring-resource-quota.adoc
 
 :_content-type: PROCEDURE
 [id="configuring-workloads_{context}"]
 = Configuring workloads with resource requests and limits
 
-[role="_abstract"]
```

Comment:
Why this removal?

Reply:
This is not required actually. This was part of Jupiter work efforts, but I received in the OCP team review comments later on that we can remove it. Hence removing it now.